src/utils/Config.php

```diff
@@ -396,6 +396,30 @@ public function getNested($key, $default = null){
 		return $this->nestedCache[$key] = $base;
 	}
 
+	public function getNestedString(string $key, string $default = null) : ?string {
+		return is_string($value = $this->getNested($key)) ? $value : $default;
```

Comment:
This needs to cast `int` and `float` to `string` (yaml may cause strings such as `4.0` to be returned as floats)